lib/components/narrative/save-trip-button.js

```diff
@@ -50,14 +52,20 @@ const SaveTripButton = ({
       disabled={buttonDisabled}
       style={buttonDisabled ? { pointerEvents: 'none' } : {}}
     >
-      <i className={icon} /> {buttonText}
+      <i className={icon} />
+      {' '}
+      <FormattedMessage id={buttonTextId} />
     </button>
   )
   // Show tooltip with help text if button is disabled.
   if (buttonDisabled) {
     return (
       <OverlayTrigger
-        overlay={<Tooltip id='disabled-save-tooltip'>{tooltipText}</Tooltip>}
+        overlay={(
+          <Tooltip id='disabled-save-tooltip'>
+            {intl.formatMessage({id: tooltipTextId})}
```

Comment:
Can `FormattedMessage` be used here instead?

Reply:
Tooltip children is another case where FormattedMessage can't be used